tools/test-popular-containers/test-docker-rootfs.py

```diff
@@ -8,12 +8,20 @@
 Test all the ext4 rootfs in the current directory
 """
 
+import os
+import sys
 from pathlib import Path
 
+# Hack to be able to import testing framework functions.
+sys.path.append(os.path.join(os.getcwd(), "tests"))  # noqa: E402
```

Comment:
I think this one is working fine?

Reply:
do you mean if we need the fix in this script? if so yes, the script doesn't work without the fix.

Reply:
yes pipeline is working, for ref. I checked it here:  https://buildkite.com/firecracker/sudan-tests/builds/10#018acb83-eee4-4b61-90be-1b97b81668a8